README.md

```diff
@@ -40,6 +41,7 @@ Test packages uploaded to the staging repos:
 
 The following inputs can be specified to override the default behavior
 
+* `repoVersion`: Version to be downloaded from the repo, by default will be the latest (v is stripped automatically if found).
```

Comment:
Nit: would it make sense to add a line saying it does nothing when `packageLocation` is `"local"`?

Reply:
True, I will add it

Reply:
Done